### ISSUE TITLE
[Winlogbeat] Language setting for event log extraction

### ISSUE THREAD

**Issue by DrewHorrigan** [elastic/beats#14209]
As per [Elastic form thread](https://discuss.elastic.co/t/forwarding-windows-event-logs-in-another-language/204067), Winlogbeat extracts Windows event logs in current system language, and this setting currently cannot be modified (some other threads suggested running the winlogbeat process as a user with the required language settings, but this does not change anything). It would be nice if the extracted log language was configurable.

The specific use case is server infrastructure with mixed system languages (in this particular case, some servers use English, others use a different language). All logs must be normalized into a single language, preferably English, and changing the system language for some servers is not an option. The problem cannot be solved by winlogbeat in its current form, and requires workarounds.

**Comment by strawgate** [comment]
This causes big problems in our environment because some users have Portuguese as their system language. This causes even things log like levels to show as duplicates (error and erro), keywords to show up twice, etc.

I can't imagine how bad this would be in an organization that has users from many locales and languages.

**Comment by standa4** [comment]
This issue also causes me some problems so I've tried to modify the code to add this functionality but I am not go developer so it is not perfect but it seems to work. I've created pull request for that but someone who has more experience with development of winlogbeat should finish my work.
Right now in works for me (I've tried to compile it in the 7.8.0 branche and it works) but I could be missing something so the review is needed.
